src/ssh/setup.py

```diff
@@ -7,7 +7,7 @@
 
 from setuptools import setup, find_packages
 
-VERSION = "1.0.1"
+VERSION = "1.0.0"
```

Comment:
Why it was changed back?

Reply:
We still have some changes to make. We want to do it in a different PR.
Changing the version here will do a new release of "az ssh" cli extension. We want to change the version once we have all the changes.